packages/jellyfish-api-core/__tests__/category/account/transferDomain.test.ts

```diff
@@ -233,7 +248,8 @@ describe('TransferDomain', () => {
             address: dvmAddr,
             amount: '3@DFI',
             domain: TransferDomainType.DVM
-          }
+          },
+          singlekeycheck: false as unknown as TransferDomainOptionalInfo
```

Comment:
singlekeycheck flag can be passed both on the RPC and/or set directly as a node config on start up. Reference: https://github.com/DeFiCh/ain/pull/2714 and https://github.com/DeFiCh/ain/pull/2709. To maintain backwards compatibility, a better fix could potentially be to pass the singlekeycheck flag directly on the node start up, which will default to skipping the singlekeycheck pipeline on transferdomain RPCs.

Reply:
easier to review as lesser changes now
[7525d04](https://github.com/BirthdayResearch/jellyfishsdk/pull/2173/commits/7525d04f86ff7cf48d3ee9b971a214fa44289193)

hmm.. we still need show case how to TD with singlekeycheck params
so added one [e7927cc](https://github.com/BirthdayResearch/jellyfishsdk/pull/2173/commits/e7927cc9842029559eee4c2763321936db1905a4)